Dockerfile

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project
+
+# Thes are optional tags used to add additional metadata into the docker image
+# These may be supplied by the pipeline in future - until then they will default
+
+ARG version=2.10-SNAPSHOT
+ARG EGERIA_BASE_IMAGE=odpi/egeria:latest
+
+# This Dockerfile should be run from the parent directory of the egeria-connector-sas-viya directory
+# ie
+# docker -f ./Dockerfile 
+
+FROM ${EGERIA_BASE_IMAGE}
```

Comment:
maybe 
FROM ${EGERIA_BASE_IMAGE}:${version} 

What you have works, but makes 'version' superfluous?

When we get around to extracting version from the code, having it seperate to the repo name probably is helpful.